dbsnp-importer/src/test/java/uk/ac/ebi/eva/dbsnpimporter/jobs/steps/processors/SubSnpCoreFieldsToVariantProcessorTest.java

```diff
@@ -306,11 +310,41 @@ public void testGenotypesWithN() throws Exception {
         assertVariantEquals(variant, processor.process(subSnpCoreFields));
     }
 
+    @Test
+    public void testFrequenciesSinglePopulation() throws Exception {
```

Comment:
I still have to add more tests to this class: variant with several frequencies, variant with frequencies AND genotypes.

Reply:
a test with an `assertNotEquals` with just one frequency of mismatch would be nice to check that the equals are working as expected.